dummy_oauth.sh

```diff
@@ -0,0 +1,12 @@
+echo " -------------- DUMMY OAUTH -------------- "
```

Comment:
This seems to be a script to be used to set up a stand-alone dev environment; should it be located in build/dev?